Documentation/Signoffs/Attachment Signoff.md

```diff
@@ -0,0 +1,82 @@
+
+## Constraints
+
+### Derived from Shall Statements
+
+- C4: The robot shall contain modules that are plug and play adaptable for different IEEE competition requirements.
+	- This constraint is applicable to the attachment subsystem because components need to have the ability to be added, removed, and or moved easily.
+
+### Derived from Socioeconomic Impacts
+
+- The components will be 3D printed to reduce cost of project. 	
+	- It is important that costs are reduced because the project is aimed at being recreated each year. Therefore, the lowest possible cost without reducing robot performance is a goal for the team. 
+		
+### Derived from Specifications
+- S1: The line sensor attachment must be designed to place the sensor between 0.125 and 0.375 inches off the ground.
+	- This is derived from manufacturing specifications for proper operation of the line sensor. 
+	
+## Buildable Schematics
+
+### Line Following Sensor
+The line following sensor comes in an 8 sensor set that is broken along a preforated line that splits the sensor into a 6 sensor set and a 2 sensor set. Therefore, an attachment will created for each. The buildable schematic for each is seen below. The attachment for both is comprised of two peices that will be connected through a screw and washer to allow for height adjustment from the minimum to maximum recommended height of 0.125 - 0.375 inches.
+
+#### 6 Sensor Set Attachment
+![Alt text](https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/attachment-signoff/Documentation/Images/Line%20Sensor%206%20Bracket%20Drawing.jpg)
+The above component is the what the physical sensor will attach to. It is designed in such a way that at the maximum height of the sensor will not rise up into the attachment. Becasue the sensor reads reflectance off the ground, if any portion of the attachment caused shadowing onto the sensor the readings could be inaccurate. Additionally, the slots cut into the side of this bracket are for wire connections to be placed and exit to the center of the robot. This component will sit into the adjustment mechanism below. 
+
+![Alt text](https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/attachment-signoff/Documentation/Images/6%20Backet%20Adjustment%20Drawing.jpg)
+The above component attaches to the chassis frame on the underside of the extruded alumnium. In the analysis section, the distance from the bottom of the extruded aluminum to the ground will be calculated to confirm that this component places the sensor at the appropriate height. The adjustment mechanism works through loosening and tightening a screw with a washer to catch the edge. The component attaches to the extruded alumnium through a T-channel bolt and a 1/4-20 screw. Similar to the height adjustment, the bolt can slide along the extuded alumnium when the screw is loosened.
```

Comment:
Could you explain further how did you get the width of the frame? Is it the same as the chassis frame?

Reply:
Do you mean height? Just checking so I can make the correct additions

Reply:
I just made edits to confirm this

Reply:
to fix